src/site/content/en/angular/route-preloading/index.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: 'Route preloading strategies in Angular'
+subhead: |
+  Preload routes ahead of time to speed up users' navigation.
+hero: route-preloading.jpg
```

Comment:
Mind adding `hero_position: bottom` to better center the image?

Reply:
Done